index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Conference Page - home</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <a class="mobile menu-h "><img class='mobile menu-h' src="Icons/menu.png"></a>
+      <a href="index.html"><div class="nav-logo"></div></a> 
+      <ul class="desktop nav-ul">
+        <li class="nav-il"><a href="about.html">about</a></li>
+        <li class="nav-il"><a href="about.html">Program</a></li>
+        <li class="nav-il"><a href="about.html">Sponsor</a></li>
+        <li class="nav-il"><a href="about.html">News</a></li>
+        <li class="nav-il"><a href="about.html">Stoic campaing</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div id="modal"class="modal mobile">
```

Comment:
- [ ] I would kindly suggest we stick to the Behance design as much as possible, currently, your design missing the social media navbar as seen in the design, kindly add it as this will show your confidence in the ability to replicate any given design.

   - N.B: please kindly implement this both on the `home page` and `about page`

Behance design 👇 

![beh1](https://user-images.githubusercontent.com/78634841/217394786-b9e45c48-a74d-44a4-90fa-4e43d97a8a9d.png)